softvis3d-frontend/src/components/scene/EventDispatcher.ts

```diff
@@ -0,0 +1,52 @@
+export class Event<T> {
```

Comment:
Maybe a separate folder "events" within the scene folder for The Dispatcher and it's Handlers?